wrappers/golang/fields/babybear/ntt/ntt.go

```diff
@@ -5,13 +5,15 @@ package ntt
 import "C"
 
 import (
-	"unsafe"
-
 	"github.com/ingonyama-zk/icicle/v2/wrappers/golang/core"
 	cr "github.com/ingonyama-zk/icicle/v2/wrappers/golang/cuda_runtime"
 	babybear "github.com/ingonyama-zk/icicle/v2/wrappers/golang/fields/babybear"
 )
 
+import (
```

Comment:
Same here

Reply:
Its a difference in gofmt versions. Even though it is in our template in two separate imports, the latest gofmt should move them to a single one. Using go version 1.20.X causes these changes and Italy is using 1.20.4